docs/source/required_experiment_files.rst

```diff
@@ -53,3 +53,24 @@ name. The files are as follows:
    required for dallinger2.js to work.
 -  dallinger.css - this contains several css classes that are used in the
    demos.
+
+Custom files
+^^^^^^^^^^^^
+
+You can specify files from outside the experiment directory that should
+be merged in using the `extra_files` function. This is a module-level
+function in the `experiment.py` file that returns a sequence of source
+and destination tuples. The source can be either a file or a directory.
+
+For example:
+
+.. code-block:: python
+
+    def extra_files():
+        return [
+            ("/home/user/stimulus.txt", "/static/stimulus.txt"),
+            ("/home/user/stimuluses", "/static/stimuluses"),
+        ]
+
+You can also provide this as a class method on the
```

Comment:
Does this need to be a function/method vs an attribute? It seems to be overcomplicating things to me.

Reply:
It being a callable makes it follow the same pattern as the extra routes option, and it allows easier composition with superclasses. If it wasn't a callable we couldn't call super() then add something.

Reply:
True, but you could do `extra_files = ParentClass.extra_files.copy().extend([...])` or whatever, but the fact that we already use the same pattern for routes is reason enough for me.

Reply:
I generally try to avoid complex logic in class scope, but sounds like we're agreed!